src/wrappers.js

```diff
@@ -489,3 +499,221 @@ wrappers[".google.protobuf.Timestamp"] = {
         return this.toObject(message, options);
     }
 };
+
+wrappers[".google.protobuf.StringValue"] = {
+    fromObject: function(object) {
+        if (object instanceof this.ctor) return object;
+
+        if (isLegacyValueObject(object)) {
+            return this.create(object);
+        }
+
+        if (typeof object === "string") {
+            return this.create({ value: object });
+        }
+
+        return this.fromObject(object);
+    },
+    toObject: function(message, options) {
+        if (message && typeof message === "object" && message && message.value) {
+            return message.value;
+        }
+
+        return this.toObject(message, options);
+    }
+};
+
+wrappers[".google.protobuf.BoolValue"] = {
+    fromObject: function(object) {
+        if (object instanceof this.ctor) return object;
+
+        if (isLegacyValueObject(object)) {
+            return this.create(object);
+        }
+
+        if (typeof object === "boolean") {
+            return this.create({ value: object });
+        }
+
+        return this.fromObject(object);
+    },
+    toObject: function(message, options) {
+        if (message && typeof message === "object" && typeof message.value === "boolean") {
+            return message.value;
+        }
+
+        return this.toObject(message, options);
+    }
+};
+
+wrappers[".google.protobuf.BytesValue"] = {
+    fromObject: function(object) {
+        if (object instanceof this.ctor) return object;
+
+        if (isLegacyValueObject(object)) {
+            return this.create(object);
+        }
+
+        if (typeof object === "string") {
+            return this.create({ value: object });
+        }
+
+        return this.fromObject(object);
+    },
+    toObject: function(message, options) {
+        if (message && typeof message === "object" && message.value) {
```

Comment:
I think we should explicitly check for the expected message.value here to avoid `Buffer.from` from throwing